hoomd/md/methods.py

```diff
@@ -647,9 +716,16 @@ class NVE(_Method):
         limit (None or float): Enforce that no particle moves more than a
             distance of a limit in a single time step. Defaults to None
 
+        manifold_constraint (hoomd.md.manifold.Manifold): Manifold constraint which is
+            used by and as a trigger for the RATTLE algorithm of this method.
+            Defaults to None
+
+        eta (float): Defines the error particles are allowed to deviate
+            from the manifold in terms of the implicit function. Defaults to 1e-5
+
     """
 
-    def __init__(self, filter, limit=None):
+    def __init__(self, filter, limit=None, manifold_constraint=None, eta=0.000001):
```

Comment:
The name `eta` is too generic here. Instead of adding a new generic-sounding parameter `eta` to all integration methods, could we instead give the tolerance as part of the manifold class? That would remove some of there generality and would make `ManifoldConstraint` a better class name.

Or at least, rename the integrator parameter from `eta` to `manifold_tolerance`.